src/includes/developers/offerings.njk

```diff
@@ -0,0 +1,214 @@
+          svg: "<path stroke-linecap='round' stroke-linejoin='round' stroke-width='2' d='M19.428 15.428a2 2 0 00-1.022-.547l-2.387-.477a6 6 0 00-3.86.517l-.318.158a6 6 0 01-3.86.517L6.05 15.21a2 2 0 00-1.806.547M8 4h8l-1 1v5.172a2 2 0 00.586 1.414l5 5c1.26 1.26.367 3.414-1.415 3.414H4.828c-1.782 0-2.674-2.154-1.414-3.414l5-5A2 2 0 009 10.172V5L8 4z' />",
+          sections: [
+            {
+              title: "Sign Up",
+              link: "https://airtable.com/shrMais1HjToBzPpE",
+              description: "Register for the Orbit beta program"
+            },
+            {
+              title: "Discord Integration",
+              link: "https://docs.orbit.love/docs/set-up-the-orbit-discord-integration",
+              description: "Read the docs on using the Discord beta integration"
+            },
+            {
+              title: "Webhooks",
+              link: "https://docs.orbit.love/docs/webhooks",
+              description: "Learn how to set up webhook notifications with Orbit"
+            },
+            {
+              title: "Search Changes",
+              link: "https://docs.orbit.love/docs/search-changes-march-2021",
+              description: "Important changes to the search interface"
+            }
+          ]
+        }
+      %}
+      {{ card(beta_program) }}
+
+      {%
+        set community_built = {
+          title: "Community Built",
+          description: "Explore and utilize open-source tooling built by members of the Orbit developer community.",
+          svg: "<path stroke-linecap='round' stroke-linejoin='round' stroke-width='2' d='M10 20l4-16m4 4l4 4-4 4M6 16l-4-4 4-4' />",
+          sections: [
+            {
+              title: "NodeOrbit",
+              link: "https://github.com/davidgs/NodeOrbit",
+              description: "<a href='https://nodered.org/' class='underline'>Node-RED</a> integrations by <a href='https://twitter.com/davidgsIoT' class='underline'>David G. Simmons</a>"
+            },
+            {
+              title: "OrbitParser",
+              link: "https://github.com/davidgs/OrbitParser",
+              description: "Parse your Orbit JSON into a CSV file in Golang by <a href='https://twitter.com/davidgsIoT' class='underline'>David G. Simmons</a>"
+            },
+            {
+              title: "Slack to Orbit Zapier Template #1",
+              link: "https://zapier.com/shared/abe6fd30302965cdaba5ac3d60c8bc19ea1ae791",
+              description: "Format Slack messages and send them to Orbit on Zapier with this template by <a href='https://twitter.com/mikermcneil' class='underline'>Mike NcNeil</a>"
+            },
+            {
+              title: "Slack to Orbit Zapier Template #2",
+              link: "https://zapier.com/shared/e3bd8574eceb8a7f53608394e5926d6952107a1e",
+              description: "Capture Slack anywhere messages and send them to Orbit as a new activity by <a href='https://twitter.com/fharper' class='underline'>Frédéric Harper</a>"
```

Comment:
```suggestion
              description: "Capture Slack messages and send them to Orbit as a new activity by <a href='https://twitter.com/fharper' class='underline'>Frédéric Harper</a>"
```

Reply:
In that case that's a copy and paste from the resources page ;)